docs_manual/source/metadata_advanced.rst

```diff
@@ -216,3 +218,40 @@ Protein Expression Profiling / Mass Cytometry Plugin
     * *Raw Data Files* and *Derived Data Files* are linked to ``{Assay Name}``
 - Used with measurement type / technology type
     * protein expression profiling / mass cytometry
+
+Universal Assay Plugin
+----------------------
+
+This plugin enables the user to define row-specific and inline links to
+iRODS collections via comments in the ``STUDY ASSAYS`` section of the ISA-Tab
+investigation file.
+
+- Internal name: ``samplesheets_assay_generic``
+- Row-specific links
+    * Place one or multiple comments starting with ``SODAR Assay Row Path``.
+    * Each comment should define one column name.
+    * Values of those columns are used to define the ``RowPath``.
+    * For example:
+        .. code-block::
+
+            STUDY ASSAYS
+            Comment[SODAR Assay Row Path 1]    Pool ID
+            Comment[SODAR Assay Row Path 2]    Extract Name
+
+        + Resulting row links:
```

Comment:
This is misaligned when the doc is rendered. I guess it should be de-indented to the previous level.

![plugin-doc](https://github.com/user-attachments/assets/93bac099-c452-4c66-8917-f0ca06c14867)

Reply:
This was intentional as it is part of the example. You are not a fan of the formatting?